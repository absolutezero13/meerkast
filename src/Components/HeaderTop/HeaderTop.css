.header-top {
  display: flex;
  height: 7vh;
  background: rgba(1, 126, 254, 255);
  color: white;
  box-shadow: 3px 3px 10px black;
  padding: 0 10%;
  align-items: center;
  justify-content: space-between;
  border-bottom: 1px solid black;
}
.header-top img {
  position: absolute;
  left: 1%;
  width: 30px;
  height: 30px;
}
.nav {
  display: flex;
  align-items: center;
  justify-content: space-between;
  width: 220px;
  font-weight: 900;
}

.nav button {
  background: rgba(66, 66, 66, 255);
  color: white;
  font-size: 1 rem;
  outline: none;
  font-weight: 600;
  height: 2rem;
  cursor: pointer;
}

@media only screen and (max-width: 600px) {
  .header-top,
  .nav button {
    font-size: 75%;
  }
  .header-top {
    padding: 0 6%;
  }
  .nav {
    width: 140px;
  }
  .header-top img {
    left: 0.3%;
    width: 15px;
    height: 15px;
  }
}
