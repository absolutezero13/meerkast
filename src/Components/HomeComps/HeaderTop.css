.header-top {
  display: flex;
  height: 10vh;
  background: rgba(1, 126, 254, 255);
  color: white;
  box-shadow: 3px 3px 10px black;
  padding: 0 10%;
  align-items: center;
  justify-content: space-between;
  border-bottom: 1px solid black;
}

.nav {
  display: flex;
  align-items: center;
  justify-content: space-between;
  width: 220px;
  font-weight: 900;
}

.nav button {
  background: rgba(66, 66, 66, 255);
  color: white;
  font-size: 1 rem;
  outline: none;
  font-weight: 900;
  height: 2rem;
  cursor: pointer;
}
