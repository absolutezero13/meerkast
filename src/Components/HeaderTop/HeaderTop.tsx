import React from "react";
import "./HeaderTop.css";
import logo from "../../images/mirket.png";

const HeaderTop: React.FC = () => {
  return (
    <div className="header-top">
      <img src={logo} />
      <h1>Meerkast</h1>
      <div className="nav">
        <p>Log in</p>
        <button>Start your Free trial</button>
      </div>
    </div>
  );
};

export default HeaderTop;
