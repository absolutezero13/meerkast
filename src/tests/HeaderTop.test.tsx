import React from "react";
import { render } from "@testing-library/react";
import HeaderTop from "../Components/HeaderTop/HeaderTop";

const { container } = render(<HeaderTop />);

// SnapShot Test
it("headertop matches snapshot", () => {
  expect(container).toMatchSnapshot();
});
// Text test

// beforeEach(() => {
//   container = document.createElement("div");
//   document.body.appendChild(container);
// });

// afterEach(() => {
//   unmountComponentAtNode(container);
//   container.remove();
//   container = null!;
// });

// it("renders with correct text content", () => {
//   act(() => {
//     render(<HeaderTop />, container);
//   });
//   expect(container).toHaveTextContent("Meerkast");
// });
