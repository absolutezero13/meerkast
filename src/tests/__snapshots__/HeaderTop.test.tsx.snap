// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`headertop matches snapshot 1`] = `
<div>
  <div
    class="header-top"
  >
    <img
      src="mirket.png"
    />
    <h1>
      Meerkast
    </h1>
    <div
      class="nav"
    >
      <p>
        Log in
      </p>
      <button>
        Start your Free trial
      </button>
    </div>
  </div>
</div>
`;
